entity-service/src/main/kotlin/com/egm/stellio/entity/service/EntityService.kt

```diff
@@ -602,6 +602,16 @@ class EntityService(
         return neo4jRepository.deleteEntity(entityId)
     }
 
+    fun deleteEntityAttribute(entityId: String, attributeName: String, contextLink: String): Pair<Int, Int> {

```

Comment:
Is return type needed ?

Reply:
No, it should be removed.

Reply:
The current return type is not very meaningful, but in a FP way of thinking, it is recommended to return something (that you can do something with of course). A better return type would be something like Either<Boolean, Exception> (to be improved later).